db/rdbms/src/main/resources/mapper/ProcessDefinitionMapper.xml

```diff
@@ -320,4 +320,174 @@
+    FROM ${prefix}PROCESS_DEFINITION pd
+    LEFT JOIN ${prefix}PROCESS_INSTANCE pi ON pd.PROCESS_DEFINITION_KEY = pi.PROCESS_DEFINITION_KEY
+    <where>
+      <if test="authorizedResourceIds != null and !authorizedResourceIds.isEmpty()">
+        pd.PROCESS_DEFINITION_ID IN
+        <foreach collection="authorizedResourceIds" item="resourceId" open="(" separator="," close=")">
+          #{resourceId}
+        </foreach>
+      </if>
+      <if test="authorizedTenantIds != null and !authorizedTenantIds.isEmpty()">
+        AND pd.TENANT_ID IN
+        <foreach collection="authorizedTenantIds" item="tenantId" open="(" separator="," close=")">
+          #{tenantId}
+        </foreach>
+      </if>
+      <include refid="io.camunda.db.rdbms.sql.ProcessInstanceMapper.searchFilter"/>
+      <if test="filter.orFilters != null and !filter.orFilters.isEmpty()">
+        AND (
+        <foreach collection="filter.orFilters" item="filter" separator=" OR ">
+          <trim prefix="(" suffix=")">
+            <trim prefixOverrides="AND">
+              <include refid="io.camunda.db.rdbms.sql.ProcessInstanceMapper.searchFilter"/>
+            </trim>
+          </trim>
+        </foreach>
+        )
+      </if>
+    </where>
+    GROUP BY pd.PROCESS_DEFINITION_ID
+    HAVING
+    SUM(CASE WHEN pi.STATE = 'ACTIVE' AND (pi.NUM_INCIDENTS = 0 OR pi.NUM_INCIDENTS IS NULL) THEN 1 ELSE 0 END) > 0
+    OR
+    SUM(CASE WHEN pi.STATE = 'ACTIVE' AND pi.NUM_INCIDENTS > 0 THEN 1 ELSE 0 END) > 0
+    ) t
+    <include refid="io.camunda.db.rdbms.sql.Commons.keySetPageFilter"/>
+    <include refid="io.camunda.db.rdbms.sql.Commons.orderBy" />
+    <include refid="io.camunda.db.rdbms.sql.Commons.paging"/>
+  </select>
+
+  <select id="processInstanceVersionStatisticsCount" parameterType="io.camunda.search.entities.ProcessDefinitionInstanceVersionStatisticsEntity" resultType="long">
+    SELECT COUNT(*)
+    FROM
+    (SELECT
+    pd.PROCESS_DEFINITION_ID,
+    pd.PROCESS_DEFINITION_KEY,
+    pd.VERSION AS PROCESS_DEFINITION_VERSION,
+    pd.NAME AS PROCESS_DEFINITION_NAME
+    FROM
+    ${prefix}PROCESS_DEFINITION pd
+    LEFT JOIN
+    ${prefix}PROCESS_INSTANCE pi
+    ON pd.PROCESS_DEFINITION_ID = pi.PROCESS_DEFINITION_ID AND pd.VERSION = pi.VERSION
+    <where>
+      <if test="authorizedResourceIds != null and !authorizedResourceIds.isEmpty()">
+        pd.PROCESS_DEFINITION_ID IN
+        <foreach item="id" collection="authorizedResourceIds" open="(" separator="," close=")">
+          #{id}
+        </foreach>
+      </if>
+      <if test="authorizedTenantIds != null and !authorizedTenantIds.isEmpty()">
+        AND pd.TENANT_ID IN
+        <foreach item="tid" collection="authorizedTenantIds" open="(" separator="," close=")">
+          #{tid}
+        </foreach>
+      </if>
+      <include refid="io.camunda.db.rdbms.sql.ProcessInstanceMapper.searchFilter"/>
```

Comment:
🔧 Same as for the parent stats, we only need ACTIVE instances, but we don't need the whole filter.